src/main/handlers/Utilities/Utilities.class.ts

```diff
@@ -128,13 +128,19 @@ class Utilities {
 
       if (
         (prerequisite.id === 'wsl' && stdOutput) ||
+        (prerequisite.id === 'ps1ExecutionPolicy' &&
+          stdOutput.includes('Unrestricted')) ||
         (prerequisite.id === 'wslUbuntu' && stdOutput.includes(': Ubuntu')) ||
         ((prerequisite.id === 'dockerDesktop' || prerequisite.id === 'dockerDesktopUbuntu') &&
           stdOutput.includes('Server: Docker Desktop'))
       ) {
         status = AppStatus.Configured
       }
 
+      if (prerequisite.id === 'ps1ExecutionPolicy' && !stdOutput.includes('Unrestricted')) {
```

Comment:
On line 114 we are already setting `let status = AppStatus.NotConfigured` then what do we need to set this again here?